docs/run_test/troubleshoot_failures.rst

```diff
@@ -0,0 +1,116 @@
+Troubleshoot Test Failures
```

Comment:
Please merge understand_results page into this one. It doesn't need a separated page to explain test status.

Reply:
https://mslisa--3528.org.readthedocs.build/en/3528/run_test/run.html